src/Umbraco.Core/Extensions/ClaimsIdentityExtensions.cs

```diff
@@ -72,5 +75,203 @@ public static string FindFirstValue(this ClaimsIdentity identity, string claimTy
 
             return identity.FindFirst(claimType)?.Value;
         }
+
+        /// <summary>
+        /// Returns the required claim types for a back office identity
+        /// </summary>
+        /// <remarks>
+        /// This does not include the role claim type or allowed apps type since that is a collection and in theory could be empty
+        /// </remarks>
+        public static IEnumerable<string> RequiredBackOfficeClaimTypes => new[]
+        {
+            ClaimTypes.NameIdentifier, //id
+            ClaimTypes.Name,  //username
+            ClaimTypes.GivenName,
+            // Constants.Security.StartContentNodeIdClaimType, These seem to be able to be null...
+            // Constants.Security.StartMediaNodeIdClaimType,
+            ClaimTypes.Locality,
+            Constants.Security.SecurityStampClaimType
+        };
+
+        public const string Issuer = Constants.Security.BackOfficeAuthenticationType;
```

Comment:
I don't like the fact that we have a public constant just proxying another public constant.

I would use `Constants.Security.BackOfficeAuthenticationType` anywhere `ClaimsIdentityExtensions.Issuer` is used.

Reply:
Ahh that's a good point, I've replaced usages of ClaimsIdentityExtensions.Issuer with Constants.Security.BackOfficeAuthenticationType